vite.config.ts

```diff
@@ -56,14 +41,7 @@ export default defineConfig(({ command }) => ({
     },
     testTimeout: isCI ? 10000 : 5000,
     setupFiles: ['test/setup.ts'],
-    browser: {
-      enabled: true,
-      name: 'chromium',
-      provider: 'playwright',
-      commands: { resizeColumn },
-      viewport: { width: 1920, height: 1080 },
-      headless: true
-    },
+    reporters: ['basic'],
```

Comment:
The default reports just spams the console, and we missed a console log because of it.